hack/metadata.schema.json

```diff
@@ -112,7 +112,11 @@
         "type": "string",
         "pattern": "^\\d+\\.\\d+\\.\\d+$|^\\d+\\.\\d+\\.\\*$"
       }
-    }
+    },
+    "customerDataAccess": {
+      "type": "boolean",
+      "description": "Wether or not this script has access to customer data"
+  }
   },
   "additionalProperties": false,
   "required": [
```

Comment:
I would say lets make it a required field to have it consistently in every script. We could have it optionally and only fail when its missing and the target cluster is access protected. But I would be strongly in favor of making it required, so its easier to build on top of it later. wdyt?